### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Dmytro-Kulyk-cpu**
- [DEMO LINK](https://github.com/Dmytro-Kulyk-cpu/layout_html-form)
- [TEST REPORT LINK](https://github.com/mate-academy/layout_html-form/compare/master...Dmytro-Kulyk-cpu:layout_html-form:develop?expand=1)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # HTML form
-Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_html-form/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_html-form/report/html_report/)
+Replace `<https://github.com/Dmytro-Kulyk-cpu>` with your Github username and copy the links to Pull Request description:
+- [DEMO LINK](https://github.com/Dmytro-Kulyk-cpu/layout_html-form)
+- [TEST REPORT LINK](https://github.com/mate-academy/layout_html-form/compare/master...Dmytro-Kulyk-cpu:layout_html-form:develop?expand=1)
```
Comment (vlad-demchuk) [inline readme.md]: fix links
you just need to replace <your_account>, and not the whole link
```
- [DEMO LINK](https://<your_account>.github.io/layout_html-form/)
- [TEST REPORT LINK](https://<your_account>.github.io/layout_html-form/report/html_report/)

File: src/index.html [hunk 2]
```diff
@@ -11,7 +11,189 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
```
Comment (vlad-demchuk) [inline src/index.html]: remove comments

File: src/index.html [hunk 3]
```diff
@@ -11,7 +11,189 @@
...
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      method="POST"
+      action="http://localhost:3000/api">
```
Comment (vlad-demchuk) [inline src/index.html]: use correct action link from task requirements

File: src/index.html [hunk 4]
```diff
@@ -11,7 +11,189 @@
...
+        <legend>Personal information</legend>
+
+        <lable class="input-row">
+          Surname:
+
+          <input
+          type="text"
+          name="surnname"
+          autocomplete="off"
+          >
```
Comment (vlad-demchuk) [inline src/index.html]: proofread your code with checklist
```suggestion
          <input
            type="text"
            name="surnname"
            autocomplete="off"
          >
```

Comment (vlad-demchuk) [reply src/index.html]: fix all code style cases

File: src/index.html [hunk 5]
```diff
@@ -11,7 +11,189 @@
...
+
+      <fieldset class="inputs-block">
+        <legend>Additional info:</legend>
+
+        <lable class="input-row">
+          Comments:
+
+          <textarea
+          name="comments">
+          </textarea>
```
Comment (vlad-demchuk) [inline src/index.html]: check example with `textarea` in checklist

File: src/style.css [hunk 6]
```diff
@@ -1 +1,36 @@
...
+
+/* body {
+  display: block;
+  margin: 8px;
+} */
+
+/* form {
+  display: block;
+  margin-top: 0 em;
+} */
```
Comment (vlad-demchuk) [inline src/style.css]: do not leave commnets

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (Dmytro-Kulyk-cpu) [reply readme.md]: It should be like this:

- [DEMO LINK](https://<Dmutro-Kulyk-cpu>.github.io/layout_html-form/)
- [TEST REPORT LINK](https://<Dmutro-Kulyk-cpu>.github.io/layout_html-form/report/html_report/)

File: src/style.css [hunk 7]
```diff
@@ -1 +1,36 @@
-/* styles go here */
+/* head {
```
Comment (vadim-ilchenko) [inline src/style.css]: Unnecessary commented code. Remove the commented head section as it's not needed.

File: src/style.css [hunk 8]
```diff
@@ -1 +1,36 @@
...
+  display: none;
+} */
+
+/* body {
+  display: block;
+  margin: 8px;
+} */
+
```
Comment (vadim-ilchenko) [inline src/style.css]: Unnecessary commented code. Remove the commented form section as it's not needed.

File: src/style.css [hunk 9]
```diff
@@ -1 +1,36 @@
...
+.input-row {
+  display: block;
+  margin-bottom: 10px;
+}
+
+.input-row:last-child {
+  margin-bottom: 0;
+}
+
+/* input[type="checkbox" i] {
```
Comment (vadim-ilchenko) [inline src/style.css]: Unnecessary commented code. Remove the commented input[type="checkbox" i] section as it's not needed.

File: src/style.css [hunk 10]
```diff
@@ -1 +1,36 @@
-/* styles go here */
+/* head {
+  display: none;
+} */
+
+/* body {
```
Comment (vadim-ilchenko) [inline src/style.css]: Unnecessary commented code. Remove the commented body section as it's not needed.

File: src/style.css [hunk 11]
```diff
@@ -1 +1,36 @@
...
+  display: block;
+  margin: 8px;
+} */
+
+/* form {
+  display: block;
+  margin-top: 0 em;
```
Comment (vadim-ilchenko) [inline src/style.css]: Incorrect unit for margin-top value. Change '0 em' to '0em'.

File: src/index.html [hunk 12]
```diff
@@ -11,7 +11,189 @@
...
+          <input
+          type="number"
+          name="age"
+          min="1"
+          max="100"
+          value="12"
+          >
+        </lable>
+
+        <lable class="input-row">
```
Comment (vadim-ilchenko) [inline src/index.html]: Change 'lable' to 'label' for proper naming of the label element.

File: src/index.html [hunk 13]
```diff
@@ -11,7 +11,189 @@
...
+          >
+        </lable>
+
+      </fieldset>
+
+
+      <fieldset class="inputs-block">
+        <legend>Registration</legend>
+
+        <lable class="input-row">
```
Comment (vadim-ilchenko) [inline src/index.html]: Change 'lable' to 'label' for proper naming of the label element.

File: src/index.html [hunk 14]
```diff
@@ -11,7 +11,189 @@
...
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      method="POST"
+      action="http://localhost:3000/api">
+
+      <fieldset class="inputs-block">
+        <legend>Personal information</legend>
+
+        <lable class="input-row">
```
Comment (vadim-ilchenko) [inline src/index.html]: Change 'lable' to 'label' for proper naming of the label element.

File: src/index.html [hunk 15]
```diff
@@ -11,7 +11,189 @@
...
+          step="1"
+          value="0"
+          >
+        </lable>
+      </fieldset>
+
+      <fieldset class="inputs-block">
+        <legend>Additional info:</legend>
+
+        <lable class="input-row">
```
Comment (vadim-ilchenko) [inline src/index.html]: Change 'lable' to 'label' for proper naming of the label element.

File: src/index.html [hunk 16]
```diff
@@ -11,7 +11,189 @@
...
+          Surname:
+
+          <input
+          type="text"
+          name="surnname"
+          autocomplete="off"
+          >
+        </lable>
+
+        <lable class="input-row">
```
Comment (vadim-ilchenko) [inline src/index.html]: Change 'lable' to 'label' for proper naming of the label element.

File: src/index.html [hunk 17]
```diff
@@ -11,7 +11,189 @@
...
+          <label>
+            <input type="radio" name="name"
+            value="no"
+            >
+            No
+          </label>
+
+        </div>
+
+        <lable class="input-row">
```
Comment (vadim-ilchenko) [inline src/index.html]: Change 'lable' to 'label' for proper naming of the label element.

File: src/index.html [hunk 18]
```diff
@@ -11,7 +11,189 @@
...
+          Full date of birth:
+
+          <input
+          type="date"
+          name="birth"
+          >
+        </lable>
+
+        <lable class="input-row">
```
Comment (vadim-ilchenko) [inline src/index.html]: Change 'lable' to 'label' for proper naming of the label element.

File: src/index.html [hunk 19]
```diff
@@ -11,7 +11,189 @@
...
+          E-mail:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          >
+        </lable>
+
+        <lable class="input-row">
```
Comment (vadim-ilchenko) [inline src/index.html]: Change 'lable' to 'label' for proper naming of the label element.

File: src/index.html [hunk 20]
```diff
@@ -11,7 +11,189 @@
...
+          What is your favorite color?
+
+          <input
+          type="color"
+          name="color"
+          >
+        </lable>
+
+
+        <lable class="input-row">
```
Comment (vadim-ilchenko) [inline src/index.html]: Change 'lable' to 'label' for proper naming of the label element.

File: src/index.html [hunk 21]
```diff
@@ -11,7 +11,189 @@
...
+
+      <fieldset class="inputs-block">
+        <legend>Personal information</legend>
+
+        <lable class="input-row">
+          Surname:
+
+          <input
+          type="text"
+          name="surnname"
```
Comment (vadim-ilchenko) [inline src/index.html]: Correct the spelling of 'surnname' to 'surname' in the input name attribute.

File: src/index.html [hunk 22]
```diff
@@ -11,7 +11,189 @@
...
+          name="cars"
+          multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </lable>
+
+        <lable class="input-row">
```
Comment (vadim-ilchenko) [inline src/index.html]: Change 'lable' to 'label' for proper naming of the label element.

File: src/index.html [hunk 23]
```diff
@@ -11,7 +11,189 @@
...
+          What time do you go to bed?
+
+          <input
+          type="time"
+          name="time"
+          step="1"
+          >
+        </lable>
+
+        <lable class="input-row">
```
Comment (vadim-ilchenko) [inline src/index.html]: Change 'lable' to 'label' for proper naming of the label element.

File: src/index.html [hunk 24]
```diff
@@ -11,7 +11,189 @@
...
+          <label>
+            <input
+            type="radio"
+            name="cats"
+            value="yes"
+            >
+            Yes
+          </label>
+          <label>
+            <input type="radio" name="name"
```
Comment (vadim-ilchenko) [inline src/index.html]: Change the input name attribute from 'name' to 'cats' to match the other radio button in the group.

File: src/index.html [hunk 25]
```diff
@@ -11,7 +11,189 @@
...
+          Name:
+
+          <input
+          type="text"
+          name="name"
+          autocomplete="off"
+          >
+        </lable>
+
+        <lable class="input-row">
```
Comment (vadim-ilchenko) [inline src/index.html]: Change 'lable' to 'label' for proper naming of the label element.

File: readme.md (hunk 1 above)
Comment (Dmytro-Kulyk-cpu) [reply readme.md]: I fixed everything. Please check

File: src/style.css (hunk 7 above)
Comment (Dmytro-Kulyk-cpu) [reply src/style.css]: I fixed everything. Please check

File: readme.md (hunk 1 above)
Comment (anstsot) [reply readme.md]: The links aren't working. please fix them

File: src/index.html [hunk 26]
```diff
@@ -11,7 +11,189 @@
...
+
+      <fieldset class="inputs-block">
+        <legend>Additional info:</legend>
+
+        <label class="input-row">
+          Comments:
+
+          <textarea
+            name="comments"
+         ></textarea>
```
Comment (anstsot) [inline src/index.html:180]:
```suggestion
          <textarea name="comments"></textarea>
```

**Review by anstsot — CHANGES_REQUESTED** [review]
Please check the links because they aren't working. If you have any questions, feel free to ask them in the chat

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]
make sure you follow the instruction

the links still aren't working
just ask for help in `fe_chat`

File: src/index.html [hunk 27]
```diff
@@ -11,7 +11,188 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <title>HTML Form</title>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      method="POST"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application">
```
Comment (vlad-demchuk) [inline src/index.html:19]:
```suggestion
    <form
      method="POST"
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
    >
```

**Review by vlad-demchuk — APPROVED** [review]

**Pull request closed without merging**